packages/chart/src/types/ChartConfig.ts

```diff
@@ -96,6 +96,7 @@ type AllChartsConfig = {
   dataFormat: DataFormat
   dataKey: string
   description: string
+  displayLineGraphAs: 'Line' | 'Bar'
```

Comment:
make this a boolean  and change the variable to convertLineToBarGraph. then put a comment next to this type that explains that this is only for when there's less than 3 values.

Reply:
remove displayLineGraphAs from this whole PR and replace with convertLineToBarGraph